lib/components/ResourceList/ResourceList.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import resources from '../../../test-data/resources.json';
+import { ResourceList } from './ResourceList';
+
+const items = resources.map((resource) => ({
+  id: resource.identifier,
+  ownerName: resource.ownerName ?? '',
+  resourceName: resource.resourceName,
+  ownerLogoUrl: resource.ownerImageUrl ?? '',
+  ownerLogoUrlAlt: 'resource.ownerName + logo',
+}));
+
+const meta = {
+  title: 'Resource/List',
```

Comment:
What do you think about having this story placed under List as `ResourceList` instead of under `Resource`? I.e.

```
  title: 'List/ResourceList',
```
I think it makes sense that this is a variant of a ListItem with a very strict purpose of rendering the same way

Reply:
Yes, that makes sense. I'll update the title.